seed/seed20.go

```diff
@@ -496,34 +628,26 @@ func (s *seed20) loadEssentialMeta(filterEssential func(*asserts.ModelSnap) bool
 	essSnaps := model.EssentialSnaps()
 	const essential = true
 
+	// create channels
+	if filterEssential != nil {
+		// LoadEssentialMeta
+		s.snapsToConsiderCh = make(chan snapToConsider, 4)
```

Comment:
would this work too?
```suggestion
		s.snapsToConsiderCh = make(chan snapToConsider, len(essSnaps))
```

Reply:
not quite because snapd might be implicit, I reworked the code somewhat